crates/nu-cli/tests/commands/mod.rs

```diff
@@ -1,5 +1,6 @@
 mod alias;
 mod append;
+mod cal;
```

Comment:
Looks like you might be missing a test file

Reply:
Ahhh shoot, you’re right.  I didn’t add any unit tests for this.  I guess I’ll have I look it over in the next day or two and see what I can do.

Reply:
We have a few commands (like `fetch`) that don't have unit tests because they work against a live system.  I'm okay with not having tests for now until we get a better way to mock or record runs.

Reply:
Though I guess with `cal`, you have the ability to ask for a known month/year so yeah, it does make sense to add some tests.

Reply:
I was thinking about this as well, I assumed I’d just add some tests for a specific year or month.  There’d be no way to add tests fo the default command since the month would change.  I was planning on looking into what it would take to add these tests today at some point.  I may be on Discord if I get stuck at some point.